CHANGELOG-3.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
```

Comment:
Should this link to kubernetes-csi docs instead?

Reply:
Fixed